### ISSUE TITLE
508/a11y: Methods & Evaluation Resources page - image missing alt text

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#341]
Page: [Methods & Evaluation Resources](https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/methods/)

The image of the project process is missing alt text. Related to issue #337. Recommend a similar fix to what #337 suggests.

Screenshot:
<img width="673" alt="Screen Shot of complex image without alt text highlighted in browser testing tool" src="https://user-images.githubusercontent.com/1093423/130137648-f9cbd4fd-0b00-452d-84da-b459f22446d3.png">
